### ISSUE TITLE
Network check should be escapable

### ISSUE THREAD

**Issue by puneetse** [clearlinux/clr-installer#635]
**Describe the solution you'd like**
In the graphical installer, the network check that should have an option to cancel/interrupt the network check before completion. 

This is useful if you know network connectivity will fail so you want to exit. Right now you're stuck until the check completes even if you know ahead of time you need to correct something. 

**Describe alternatives you've considered**
Breath deeply and wait.

**Additional context**
The network check can happen abruptly when entering the `Select Additional Bundles` screen.

**Comment by mdhorn** [comment]
@puneetse If I recall from the initial implementation, we were having an issue between the go threads and GTK to make a Cancel button work, but we can explore this again.

When we enabled the ISO images with offline content, meaning no network connection needed, we had to make q trade-off between size and network availability. All of the default bundles are available without needing a network, but selecting any of the Optional bundles requires a network.

A couple of other possible solutions:
- Have a pop-up when you enter the Optional Bundle page that states a network test needs to run and pass in order to select additional bundles. This could easily have OK and Cancel buttons.
- Not start the network check until one of the additional bundles is actually selected. This would allow easy browsing of the additional bundles without suffering the network check until one is selected.
- Combination of both of the above.

Regardless, however, we implement this for the graphical interface, we should do the same for the text-based for consistency.
